crates/sui-cluster-test/src/wallet_client.rs

```diff
@@ -25,8 +25,7 @@ impl WalletClient {
         let key = cluster.user_key();
         let address: SuiAddress = key.public().into();
         let wallet_context = new_wallet_context_from_cluster(cluster, key)
-            .instrument(info_span!("init_wallet_context_for_test_user"))
-            .await;
+            .instrument(info_span!("init_wallet_context_for_test_user")).await;
```

Comment:
i think we can drop the await here since the function is not async anymore

Reply:
ues ```let wallet_context = new_wallet_context_from_cluster(cluster, key);``` directly.